plugin.rb

```diff
@@ -613,6 +613,26 @@ class ::ListableTopicSerializer
     options[:refresh_stream] = true if old_category_allows != new_category_allows
   end
 
+  query = "
+    WITH x AS (SELECT
+      u.id user_id,
+      COUNT(DISTINCT ua.id) AS solutions
+      FROM users AS u
+      INNER JOIN user_actions AS ua ON ua.user_id = u.id AND ua.action_type = 15 AND COALESCE(ua.created_at, :since) > :since
```

Comment:
Is there a constant we can use for that `15`?

Reply:
Yeah, I will switch it to user `UserAction::SOLVED` which is _so weird_ to have in core...